geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientNotifier.java

```diff
@@ -2157,4 +2054,20 @@ public void run2() {
       }
     }
   }
+
+  public static CacheClientNotifierProvider singletonProvider() {
+    return CacheClientNotifier::getInstance;
+  }
+
+  @SuppressWarnings("unused")
```

Comment:
Is this a test only method if "unused"? Should it be annotated as such?

Reply:
Yes, these should be VisibleForTesting. Thanks!